canvas/models.py

```diff
@@ -159,6 +164,30 @@ class Meta:
     def get_token_uses(self):
         return [get_token_use(self.user, tup['id']) for tup in self.course.token_use_options.values('id')]
 
+    @property
+    def is_verified(self):
+        return self.status == 'VERIFIED'
+
+    @property
+    def is_blocked(self):
+        return self.status == 'BLOCKED'
+
+    def verify(self):
+        self.status = 'VERIFIED'
+        self.save()
+
+    def block(self):
+        self.status = 'BLOCKED'
+        self.save()
+
+    def unregister(self):
+        self.status = 'UNREGISTERED'
+        self.save()
+
+    def unverify(self):
+        self.status = 'PENDING_VERIFICATION'
+        self.save()
```

Comment:
Maybe combine into one method, that takes in an input on what to save as. so something like `change_status(self, status)`. Sample call would be like `change_status(STATUS["UNREGISTERED"])`

Reply:
I dont think we should do it as I just notice that I have to make STATUS array exported to use it in course_admin or other views.